.pre-commit-config.yaml

```
@@ -34,6 +34,13 @@ repos:
   - id: check-merge-conflict
   - id: debug-statements
     language_version: python3
+- repo: https://github.com/adrienverge/yamllint.git
+  rev: v1.23.0
+  hooks:
+  - id: yamllint
+    files: \.(yaml|yml)$
+    types: [file, yaml]
```

I'm pretty sure this should be
```suggestion
    types: [yaml]
```

---

See https://github.com/adrienverge/yamllint/blob/master/.pre-commit-hooks.yaml -- that is the recommended setup. Keep in mind that these work with AND and not OR.